shaazk/Free-Mentors
# #168163530 update endpoints

#### What does this PR do?
- it updates a sign in, signup and views all mentors endpoints
#### Description of Task to be completed?
- a user can create an account
- a user can login 
- a user can view all mentors

#### How should this be manually tested?
 -clone free mentors on your computer and open the older in your VS Code or any other html IDE. in your terminal, run nodemon by typing npm start and then open your postman and run http://localhost:5000/api/v1/signup under POST function and in body, insert credentials and click on send same as other endpoints respectively
- in your VSCode after running nodemon, you can click on ctrl + the link in the terminal where you see server running on http://localhost:3000
#### What are the relevant pivotal tracker stories?
- [#168163530](https://www.pivotaltracker.com/story/show/168163530)
